docsets/Ant_Design/docset.json

```diff
@@ -1,13 +1,17 @@
 {
     "name": "Ant Design",
-    "version": "1.0.1",
+    "version": "3.3.0",
     "archive": "Ant_Design.tgz",
```

Comment:
You need to update the archive at this path as well.

Reply:
Done.